src/main/kotlin/silentbombers/commerce/member/adapter/output/MemberEntity.kt

```diff
@@ -0,0 +1,25 @@
+package silentbombers.commerce.member.adapter.output
+
+import jakarta.persistence.*
+
+@Entity
+class MemberEntity(
+    id: Int,
+    name: String,
+    phoneNumber: String?,
+    address: String?,
+) {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    val id: Int = id
+
+    @Column
+    val name: String = name
+
+    @Column
+    val phoneNumber: String? = phoneNumber
+
+    @Column
+    val address: String? = address
+}
```

Comment:
```suggestion
@Entity
class MemberEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Int,
    @Column
    var name: String,

    @Column
    var phoneNumber: String,

    @Column
    var address: String,
)
```
이렇게 바꿔보는거 어떨까여? 변경될 수 있는 것들은 var로 바꾸는게 좋을거 같아요